opentelemetry-otlp/Cargo.toml

```diff
@@ -34,7 +34,7 @@ opentelemetry-proto = { version = "0.28", path = "../opentelemetry-proto", defau
 tracing = {workspace = true, optional = true}
 
 prost = { workspace = true, optional = true }
-tonic = { workspace = true, optional = true }
+tonic = { workspace = true, default-features = false, optional = true }
```

Comment:
is it required to state defaut-features=false here, given it is already in the workspace?